### ISSUE TITLE
Screen reader hidden text can overlap the right margin of the screen on iOS

### ISSUE THREAD

**Issue by PeterWooster** [Automattic/_s#247]
The screen reader hidden text field can overlap the right margin of the screen if the toggle is close to the right margin and narrower than the .screen-reader-text.skip-link.

This causes a narrow margin of background colour to the right of the the screen that the user can expose by scrolling to the left.  As the field is invisible and there is no web inspector in iOS, finding the cause of this is next to impossible.  

In my case the toggle is 32px wide and the margin is 2% or 6px for a total of 38px. The screen-reader-text is 43px wide resulting in a window of 325px or a 5px margin to the right of the screen on a portrait iPhone.

To fix this I added:

``` css
.screen-reader-text.skip-link {
    right:0;
}
```

but this is not a general fix as it wouldn't work for a small left justified toggle.  A better fix might be to document that the toggle should be at least 43px wide, or make the screen reader text narrower.
